src/controllers/shopController.ts

```diff
@@ -8,32 +7,27 @@ export const createShop = async (req: Request, res: Response) => {
     shopDescription,
```

Comment:
we can consider making these exclusive types to maybe help us with organization and stuff. also these types can be helpful for frontend to backend.